src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -871,4 +866,42 @@ public static void warnOnNonIlluminaReadGroups(final SAMFileHeader readsHeader,
             logger.warn("This tool has only been well tested on ILLUMINA-based sequencing data. For other data use at your own risk.");
         }
     }
+
+    /**
+     * Returns true iff we can make instances of this class.
+     */
+    public static boolean canMakeInstances(final Class<?> clazz) {
```

Comment:
I think these new methods (`canMakeInstances()` and `makeInstances()`) should live in a separate `ClassUtils` class, instead of being buried within the general `Utils` class.